packages/web3/test/e2e/get_balance.test.ts

```diff
@@ -69,7 +69,18 @@ describe(`${getSystemTestBackend()} tests - getBalance`, () => {
 			format: [FMT_NUMBER.BIGINT, FMT_NUMBER.HEX, FMT_NUMBER.STR],
 		}),
 	)('getBalance', async ({ block, format }) => {
-		const result = await web3.eth.getBalance(getE2ETestAccountAddress(), blockData[block], {
+		let blockOrTag = blockData[block];
+		if (block === 'blockHash' || block === 'blockNumber') {
+			/**
+			 * @NOTE Getting a block too far back in history
+			 * results in a missing trie node error, so
+			 * we get latest block for this test
+			 */
+			const b = await web3.eth.getBlock('finalized');
+			blockOrTag = block === 'blockHash' ? String(b.hash) : Number(b.number);
```

Comment:
```suggestion
			blockOrTag = (block === 'blockHash') ? String(b.hash) : Number(b.number);
```
small change for readability

Reply:
prettier will remove the brackets